packages/utilities/psammead-styles/src/font-styles.js

```diff
@@ -0,0 +1,44 @@
+import * as fonts from './font-families';
+
+export const getSansRegular = service =>
+  fonts[service] && fonts[service].sansRegular;
+
+export const getSansItalic = service => {
+  if (!fonts[service]) {
+    return undefined;
```

Comment:
Usually i would prefer returning null, but im not too bothered

Reply:
Me too. @sadickisaac Can we replace them? Otherwise lgtm :)

Reply:
I was thinking more of what would `fonts['service']` return given and invalid `service` name. It would be `undefined` not `null`. Changed it though.